cloud_functions/token_refresh/streams/strava.py

```
@@ -1,113 +1,119 @@
+        )
+        # Create the document with default values
+        strava_ref.set(new_tokens, merge=True)
+        return
+
+    strava_ref.update(
+        {
+            "accessToken": new_tokens["accessToken"],
+            "refreshToken": new_tokens["refreshToken"],
+            "expiresAt": new_tokens["expiresAt"],
+            "lastTokenRefresh": new_tokens["lastTokenRefresh"],
+        }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)